templates/page/grande-marche-europe/grande-marche-europe.html.twig

```diff
@@ -0,0 +1,91 @@
+{% extends 'base.html.twig' %}
+{% block page_title 'La Grande marche' %} 
+{% block meta_description 'Aujourd\'hui le combat pour l\'égalité réelle femme / homme est culturel. 
+Les lois existent, c\'est maintenant aux mentalités d\'évoluer. 
+Ce sera la grande cause nationale du quinquennat d\'Emmanuel Macron.' %} 
+{% block canonical_url url('page_grande_marche_europe') %} 
+{% block header '' %}
+{% block banner '' %}
+{% block javascripts %} {{ parent() }}
```

Comment:
Please change indent inside `javascripts` block and move `{{ parent() }}` to new line.